### PR TITLE
feat(tracing): Document integration-specific default sampling context for Python

### PR DESCRIPTION
**Pull request by lobsterkatie**
This documents the changes made in https://github.com/getsentry/sentry-python/pull/888, https://github.com/getsentry/sentry-python/pull/906, and https://github.com/getsentry/sentry-python/pull/916 - adding information to the default sampling context passed to `traces_sampler` for the `aiohttp`, `asgi`, `aws`, `bottle`, `celery`, `django`, `falcon`, `flask`, `gcp`, `pyramid`, `rq`, `tryton`, and `wsgi` integrations to the Python SDK. (Some of those didn't directly have it added but rely on the `wsgi` integration and therefore get it for free.)

### CODE REVIEW

File: src/includes/performance/default-sampling-context/python.aiohttp.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<PlatformContent includePath="performance/default-sampling-context-platform" />
+
+The AIOHTTP integration adds the incoming request object:
+
+```python
+{
+  "aiohttp_request": aiohttp.web_request.Request
```
Comment (untitaker) [inline src/includes/performance/default-sampling-context/python.aiohttp.mdx:7]: Can we change this to `<aiohttp.web_request.Request>` or something that indicates we are passing an instance. Likewise with the others.

**Review by untitaker — APPROVED** [review]
We still need to release the sdk to merge these docs. @lobsterkatie can you try this on your own?

**Comment by lobsterkatie** [comment]
> We still need to release the sdk to merge these docs. @lobsterkatie can you try this on your own?

Thanks - meant to add a comment to the description saying as much, and have now done so.

File: src/includes/performance/default-sampling-context/python.aiohttp.mdx (hunk 1 above)
Comment (lobsterkatie) [reply src/includes/performance/default-sampling-context/python.aiohttp.mdx:7]: Sure. (I'll admit it's more obvious with the primitives.)

Comment (untitaker) [reply src/includes/performance/default-sampling-context/python.aiohttp.mdx:7]: Yeah, I get that this is probably convention in Typescript but in Python this notation hasn't really caught on yet.

**Pull request merged**